osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/ScheduledTask.java

```diff
@@ -9,170 +9,34 @@
 
 import java.io.Serializable;
 import java.sql.Timestamp;
-import java.util.Date;
-import java.util.Objects;
 
-import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Table;
 
-import org.apache.commons.lang3.StringUtils;
 import org.hibernate.annotations.Type;
-
-import org.opensmartgridplatform.domain.core.valueobjects.ScheduledTaskStatusType;
-import org.opensmartgridplatform.shared.domain.entities.AbstractEntity;
 import org.opensmartgridplatform.shared.infra.jms.DeviceMessageMetadata;
 
 @Entity
 @Table(name = "scheduled_task")
-public class ScheduledTask extends AbstractEntity {
-
-    /**
-     * Serial Version UID.
-     */
-    private static final long serialVersionUID = -77027372763748726L;
-    private static final int MAX_ERROR_MESSAGE_LENGTH = 255;
-
-    @Column(length = 255)
-    private String domain;
-
-    @Column(length = 255)
-    private String domainVersion;
-
-    @Column(length = 255)
-    private String correlationUid;
-
-    @Column(length = 255)
-    private String organisationIdentification;
+public class ScheduledTask extends AbstractScheduledTask {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS2160)

Reply:
Not needed, equals is based on correlationUid only.